Sources/MapKitSearchViewController.swift

```diff
@@ -739,11 +783,32 @@ extension MapKitSearchViewController: CLLocationManagerDelegate {
         mapView.setCenter(location.coordinate, animated: true)
         manager.stopUpdatingLocation()
     }
+    
+    public func locationManager(_ manager: CLLocationManager, didFailWithError error: Error) {
+        print(error)
+    }
+    
 }
 
 // MARK: - Protocol
 public protocol MapKitSearchDelegate {
+    
+    @available(*, deprecated, message: "Other protocol functions provide finer detail on user action")
     func mapKitSearch(_ mapKitSearchViewController: MapKitSearchViewController, mapItem: MKMapItem)
+
```

Comment:
LGTM! Would it make sense to keep the controller / self as the first parameter, just to keep the API similar to Apple's. https://developer.apple.com/documentation/mapkit/mkmapviewdelegate. 

If I did patch this in, the version would have to be major (2.X) because there's some breaking changes (unless you can make the new delegate methods optional).